sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -1281,6 +1281,9 @@ class MapboxMap internal constructor(
    * In most cases should not be called directly.
    */
   override fun gesturesPlugin(function: (GesturesPlugin.() -> Any?)): Any? {
+    requireNotNull(gesturesPlugin?.get()) {
```

Comment:
Can we throw explicit exception here? I'm not even sure what type of exception is thrown because of `requireNotNull`. IMHO we could use even create our own exception type like `InvalidViewPluginHostException` we have